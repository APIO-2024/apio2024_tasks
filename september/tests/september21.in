txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 0 1 0 4 2 6 4 5
8 9 5 7 3 4 1 6 2
10 1
0 1 2 0 3 1 6 5 2
4 9 8 5 7 3 6 2 1
10 1
0 0 2 0 0 4 6 0 6
8 7 5 1 2 3 9 6 4
10 1
0 1 2 2 2 1 3 0 4
8 5 6 7 4 3 9 2 1
10 1
0 0 0 0 3 5 6 0 5
1 9 4 8 7 2 6 3 5
10 1
0 1 2 1 0 1 3 5 1
4 7 6 3 8 5 2 9 1
10 1
0 0 2 2 3 0 3 0 2
9 5 4 6 8 7 3 2 1
10 1
0 0 2 2 0 3 5 7 7
8 9 6 3 7 5 4 1 2
10 1
0 0 0 2 0 3 0 3 5
9 6 7 5 4 1 2 8 3
10 1
0 0 0 0 1 1 4 1 5
3 6 8 7 9 5 2 4 1
10 1
0 1 2 1 3 2 6 2 1
4 3 5 7 9 8 6 2 1
10 1
0 1 2 2 0 4 5 5 3
7 3 9 8 5 6 4 1 2
10 1
0 0 0 3 4 5 5 4 5
1 2 7 8 9 6 4 5 3
10 1
0 1 1 0 4 0 6 7 2
5 8 9 4 3 2 1 7 6
10 1
0 1 0 1 0 1 1 7 0
6 4 5 3 8 2 7 1 9
10 1
0 1 0 0 1 1 0 3 4
5 6 9 4 7 2 8 1 3
10 1
0 0 0 2 4 4 6 7 1
8 5 3 9 7 1 6 4 2
10 1
0 0 1 0 4 2 3 2 4
6 2 9 5 8 7 4 1 3
10 1
0 1 2 2 2 0 5 2 1
9 4 6 3 8 7 2 5 1
10 1
0 0 0 3 1 0 1 1 4
9 7 6 5 8 2 4 1 3
40 1
0 0 2 2 0 0 4 2 8 1 3 9 9 9 10 5 6 5 6 13 5 2 2 0 2 1 19 7 25 23 6 21 17 18 7 6 34 3 32
24 39 12 38 35 31 26 14 20 15 30 16 32 13 29 33 27 36 10 19 11 23 28 25 21 37 22 7 34 9 4 1 18 8 3 2 5 6 17
40 1
0 1 0 2 3 3 2 1 3 8 0 11 6 1 14 5 14 3 4 1 19 21 2 21 1 5 2 0 2 19 5 19 6 10 34 35 33 28 38
17 29 27 9 31 30 26 20 39 15 38 36 12 37 32 24 7 25 33 16 22 19 21 18 23 35 4 5 2 11 13 14 6 3 34 28 10 8 1
40 1
0 0 1 2 1 5 6 0 8 1 5 8 5 4 4 5 10 3 12 4 13 10 13 18 3 6 4 11 1 4 7 14 15 0 24 6 26 9 0
20 23 30 35 36 21 27 25 13 16 32 22 14 34 29 39 17 31 33 37 19 7 28 26 24 11 12 6 38 5 10 15 4 3 18 1 9 8 2
40 1
0 1 0 1 1 1 0 0 0 2 6 0 12 2 9 15 9 8 18 19 5 1 14 18 5 10 3 16 19 11 11 10 14 13 7 13 19 13 0
30 37 36 29 33 23 22 28 34 14 24 4 25 39 27 20 17 31 32 26 3 38 16 10 21 35 5 11 2 6 13 15 7 19 9 1 12 8 18
40 1
0 0 1 2 0 1 4 1 0 1 3 10 11 8 3 5 6 1 5 4 10 19 1 8 14 8 6 21 11 5 15 10 32 0 34 21 22 11 30
29 18 24 36 27 23 28 35 20 26 31 34 25 14 39 15 16 7 12 9 38 32 13 33 37 30 11 4 22 8 17 21 3 10 6 19 5 1 2
100 1
0 0 1 3 4 4 4 2 0 2 5 3 6 13 8 8 3 0 0 11 17 8 15 0 6 20 10 5 14 17 22 16 12 26 10 11 35 31 37 16 21 9 31 6 13 2 12 24 12 17 50 39 45 1 45 2 1 52 20 34 53 15 31 7 58 2 15 43 68 59 33 36 0 52 69 43 49 12 23 17 30 34 75 83 76 69 6 5 16 18 77 31 6 92 80 95 76 38 36
18 74 70 62 97 90 93 44 55 51 81 59 63 89 48 41 87 56 40 94 66 82 19 84 21 57 85 88 42 27 86 32 76 25 96 29 9 60 98 73 78 95 71 91 50 67 72 80 77 64 34 49 99 26 65 92 46 14 54 7 28 16 30 83 75 20 17 69 61 68 33 47 36 43 79 38 45 24 53 23 15 58 12 11 31 13 6 5 52 22 39 37 35 8 3 1 4 2 10
1000 1
0 0 1 2 1 1 1 5 5 1 8 11 2 6 5 6 1 4 3 7 7 2 16 23 16 1 22 14 9 12 23 14 28 7 0 5 2 20 8 6 20 17 19 24 39 37 12 29 40 6 35 6 24 27 30 17 4 45 14 46 48 27 32 10 48 51 4 37 45 57 58 19 61 71 62 71 34 39 42 38 40 11 0 36 41 62 25 8 25 73 60 2 4 48 44 70 67 89 19 53 98 49 12 97 29 8 77 56 11 100 100 31 25 17 29 55 8 69 5 117 110 64 66 122 105 87 12 7 50 37 20 31 57 81 113 3 130 101 46 34 101 132 30 69 139 8 55 40 109 29 130 49 9 55 102 129 155 142 136 133 73 74 142 150 68 33 138 81 70 130 134 114 73 40 31 136 31 98 14 26 66 116 176 0 140 109 144 140 11 129 30 24 170 113 43 20 47 12 143 149 155 152 173 132 90 113 13 34 159 54 163 166 115 165 72 57 76 39 1 129 99 4 98 38 189 126 49 161 101 33 91 116 148 221 110 75 157 144 24 166 161 209 93 55 175 231 45 125 237 178 59 11 90 39 30 72 148 88 231 58 164 128 25 128 129 104 68 176 177 39 60 5 259 61 24 120 42 156 49 224 192 8 147 11 175 69 68 220 236 135 191 3 62 199 247 79 49 278 226 189 142 73 8 242 232 151 67 22 131 66 130 134 93 164 174 204 271 188 246 292 26 16 197 0 126 44 157 77 115 269 237 160 90 223 281 170 246 43 106 92 272 148 326 147 163 100 80 51 227 98 51 293 221 114 332 274 91 271 320 102 118 121 74 179 251 230 69 161 265 221 321 282 346 67 312 345 100 124 301 233 273 245 85 64 319 340 114 169 219 159 319 246 10 149 237 316 237 385 53 379 21 26 335 272 324 357 261 244 359 263 221 395 200 5 137 77 28 251 86 37 168 326 360 214 382 398 392 213 32 104 114 260 276 32 145 3 370 42 263 425 31 39 327 348 44 166 218 160 286 336 119 184 183 370 389 216 171 169 246 26 293 345 396 362 398 76 409 463 444 36 373 447 164 23 346 215 258 281 288 113 471 273 399 370 253 212 112 35 62 56 213 236 350 214 459 101 332 297 205 317 97 96 287 320 249 149 49 68 467 253 116 241 218 455 175 481 330 465 211 272 309 477 241 133 64 432 353 261 292 70 399 157 69 338 405 91 0 265 349 452 238 268 247 119 504 114 219 429 181 533 442 464 550 188 131 59 215 514 521 517 151 475 300 396 518 343 96 253 367 217 387 165 368 456 240 300 46 387 185 463 580 337 327 564 284 159 292 33 573 487 171 428 241 166 334 145 501 456 541 39 280 376 326 193 577 140 26 379 389 111 27 474 208 258 430 493 555 130 50 5 189 539 476 435 150 95 56 195 368 206 121 390 102 1 195 97 607 302 318 93 393 267 309 214 494 6 68 436 189 639 69 327 429 240 431 371 183 156 483 400 482 301 232 48 269 243 9 650 475 594 34 659 423 35 314 618 651 233 106 315 85 178 203 274 108 584 516 664 369 599 86 579 591 572 107 64 7 344 598 677 303 591 634 3 118 270 54 245 472 142 131 573 422 508 135 590 146 705 417 127 713 230 457 470 120 581 183 83 61 30 217 665 124 194 164 514 290 57 510 32 737 420 187 396 58 443 20 433 541 741 649 675 584 102 707 702 688 219 78 168 744 600 399 754 190 30 465 110 140 650 570 657 457 322 119 389 153 373 668 25 672 473 536 626 2 164 785 186 161 749 355 90 455 156 356 26 672 263 365 529 584 567 716 205 622 558 715 590 7 515 625 555 263 46 305 544 476 17 75 177 541 453 404 108 678 221 25 219 478 793 672 7 463 301 9 787 567 564 63 763 821 457 321 25 258 619 53 159 421 369 151 94 57 495 470 349 639 118 429 201 728 722 848 101 618 412 60 562 483 490 167 87 325 114 835 567 365 745 567 675 603 422 841 37 314 10 503 119 424 78 711 182 547 498 695 617 311 797 150 9 285 594 103 559 94 458 468 854 390 288 886 520 492 716 82 823 175 552 900 496 784 209 800 836 36 208 78 913 359 851 152 257 581 448 764 169 881 783 740 815 250 378 873 912 577 747 460 665 697 398 519 24 14 29 8 417 163 718 208 464 943 229 870 563 365 638 590 7 700 584 191 427 453 808 207 63 577 467 55 513 582 455 523 45 854 293 591 546 221 74 797 356 696 609 274 98 494 982 703
768 566 936 942 608 277 403 375 641 995 624 794 843 266 961 694 932 364 257 712 927 706 52 887 907 381 757 909 239 538 680 698 885 252 799 948 497 824 526 578 892 254 733 488 792 849 636 643 727 767 895 795 158 714 753 576 769 616 832 666 313 537 958 845 957 419 691 779 645 806 984 949 525 983 937 530 172 962 484 950 926 652 523 549 869 941 759 904 973 468 486 527 673 397 837 331 353 179 883 742 262 210 593 558 683 522 620 840 480 896 917 228 401 212 601 910 809 466 524 559 485 630 796 512 766 343 781 855 123 307 718 776 775 333 615 807 828 182 206 534 710 735 203 435 723 864 922 719 86 882 805 830 921 540 489 418 739 575 416 642 84 954 605 804 934 925 951 879 771 731 264 411 394 938 402 347 963 916 597 653 501 721 930 505 445 758 819 708 663 970 924 542 788 222 198 901 415 709 701 469 289 971 363 633 285 777 898 646 384 906 991 686 358 352 648 391 953 296 861 304 351 528 631 388 841 751 621 992 196 789 947 270 662 868 928 323 279 446 960 554 791 458 967 829 154 688 667 827 880 444 862 770 611 715 628 407 531 565 793 713 863 595 724 627 478 592 614 334 943 355 846 437 671 635 920 339 472 987 939 717 746 454 477 315 810 748 310 699 428 640 856 761 572 518 532 940 859 169 162 783 772 833 990 697 815 256 441 881 875 693 695 261 647 732 543 660 443 393 383 780 21 338 583 865 556 499 235 661 243 774 893 622 919 305 570 797 299 335 964 720 743 690 786 730 610 913 637 728 137 599 736 878 539 223 563 510 327 408 589 798 232 386 291 838 244 817 217 180 141 225 511 354 588 300 867 978 361 692 197 18 740 944 562 778 295 985 897 649 734 619 989 821 613 502 242 604 452 553 836 763 905 801 586 127 374 839 986 665 15 725 508 413 729 803 899 988 146 107 579 470 410 111 506 423 344 889 603 602 174 848 872 430 500 460 547 787 656 850 440 972 360 545 974 752 65 902 209 560 816 965 612 744 521 704 544 884 980 447 857 247 298 945 923 420 491 448 241 999 681 493 309 813 80 584 981 474 76 427 765 773 915 372 213 173 283 623 873 535 548 507 607 955 195 434 332 267 36 281 834 47 482 424 705 703 376 644 598 931 426 825 658 322 997 722 186 755 785 822 492 519 479 968 494 311 43 946 894 356 514 762 707 966 629 461 654 979 236 582 977 406 317 536 550 258 687 952 606 75 564 377 726 747 577 877 782 93 756 674 853 83 874 429 912 371 147 812 517 275 104 378 790 903 340 847 425 268 533 996 88 935 187 293 278 85 54 153 969 574 811 432 306 513 194 876 92 414 496 473 255 844 684 200 218 569 638 516 214 750 422 274 959 820 392 366 74 993 208 858 271 814 273 890 230 78 871 515 205 288 118 908 741 600 436 160 192 370 367 657 280 438 282 802 976 870 573 696 234 625 557 400 368 818 998 71 380 116 350 465 860 82 405 509 866 994 449 348 702 823 341 124 103 302 379 911 63 240 561 933 571 202 398 808 591 439 689 914 835 800 975 678 490 193 421 330 325 587 685 184 286 108 412 826 122 634 585 32 982 455 382 451 329 450 342 664 260 128 546 596 891 956 324 738 312 655 303 269 784 189 201 626 321 567 224 207 476 669 152 229 301 929 711 64 149 395 675 326 851 369 96 215 852 764 95 365 263 617 854 211 387 453 700 145 682 151 245 676 13 248 529 504 568 233 349 183 250 337 745 62 520 190 737 668 650 55 475 26 176 227 842 581 357 886 639 888 177 456 503 178 27 41 284 259 749 417 580 716 253 555 70 760 290 679 463 498 185 216 156 226 220 618 495 94 481 135 590 191 297 249 221 677 487 308 90 79 181 459 431 115 287 125 294 609 112 58 22 167 99 918 831 467 251 148 754 409 670 168 105 396 59 44 114 66 552 632 42 72 119 51 56 10 336 389 318 651 457 106 433 314 161 113 237 73 170 109 133 219 35 316 138 131 171 265 471 462 61 551 900 276 139 404 541 672 204 165 199 373 38 134 188 175 57 20 126 464 359 129 390 238 97 17 68 328 120 157 164 77 150 140 659 144 87 155 19 143 345 67 483 594 166 34 102 30 346 159 101 4 136 49 130 50 442 272 320 48 12 362 29 7 9 163 69 399 117 11 81 121 40 33 142 98 132 385 292 3 31 110 45 28 319 39 8 246 100 53 14 231 89 25 5 24 23 91 16 6 60 1 37 46 2
100 1
0 0 0 0 1 1 1 0 7 1 7 3 2 9 8 11 16 1 0 11 18 5 12 0 2 23 23 1 11 0 23 19 20 12 18 8 35 12 37 1 8 6 10 32 7 6 23 11 42 33 0 34 36 20 6 53 13 38 2 48 27 8 12 32 61 60 41 23 68 63 10 0 17 23 6 23 53 54 60 23 47 62 65 44 56 43 31 56 54 56 76 85 92 43 13 69 65 76 81
14 24 52 90 57 73 96 93 22 71 78 91 29 9 28 72 89 99 54 55 25 97 34 39 92 87 69 86 26 21 82 5 51 95 31 15 58 75 30 4 50 37 80 83 38 88 74 64 62 94 81 98 77 65 13 49 17 40 42 70 43 76 68 79 47 85 66 60 35 61 27 84 45 44 33 46 63 67 32 41 56 19 20 23 18 16 48 59 6 2 53 10 36 12 8 3 11 7 1
40 1
0 0 1 1 0 1 2 4 2 3 5 9 12 2 13 9 12 14 18 2 14 18 8 14 7 5 25 7 11 0 18 21 11 27 31 26 23 22 15
28 34 38 16 29 35 31 24 20 22 32 30 10 3 6 33 39 11 19 37 27 36 21 18 15 17 13 23 26 14 8 12 5 25 7 4 1 9 2
40 1
0 0 2 2 1 5 1 3 2 5 4 3 9 7 11 4 1 11 15 3 4 3 21 5 20 24 4 19 13 29 10 13 27 15 29 18 15 6 8
35 14 12 26 32 39 31 22 34 8 28 16 23 37 24 36 7 19 30 33 38 6 21 25 10 27 17 29 18 20 5 15 4 11 3 1 13 9 2
40 1
0 1 2 2 1 4 6 0 3 2 3 3 9 1 4 9 14 6 7 14 1 16 0 8 14 14 18 18 1 3 22 17 16 26 33 12 27 16 28
5 10 11 37 39 29 38 35 28 33 15 31 27 19 18 30 25 20 21 32 34 23 7 26 36 12 24 6 13 22 16 17 4 14 9 2 3 8 1
40 1
0 0 1 1 0 1 1 2 1 3 9 6 3 8 12 2 16 6 5 16 0 8 4 11 19 6 26 2 12 29 16 26 16 19 28 10 16 21 21
23 7 22 27 32 38 33 39 36 15 35 37 25 28 18 13 20 10 30 34 29 24 17 31 19 4 26 5 21 14 12 6 3 11 8 16 9 2 1
40 1
0 1 0 3 2 1 2 1 6 7 9 7 3 13 11 12 16 2 1 1 13 0 15 16 4 4 4 23 15 11 21 15 20 17 1 25 3 19 3
26 34 35 29 38 18 14 22 5 31 33 20 24 28 17 36 27 16 37 30 8 19 12 23 21 39 25 32 15 11 9 6 13 3 10 4 7 2 1
10 1
0 1 2 0 1 3 2 0 7
4 6 5 8 3 9 7 1 2
10 1
0 0 0 1 4 1 4 1 2
8 5 3 7 6 4 9 2 1
10 1
0 1 2 3 3 0 3 6 1
7 9 4 3 8 6 5 1 2
10 1
0 0 0 3 0 2 2 6 6
5 8 7 1 3 9 4 2 6
10 1
0 0 0 0 0 4 4 3 8
5 7 9 6 8 1 2 4 3
10 1
0 0 2 3 2 1 0 6 2
5 4 3 9 2 8 7 6 1
10 1
0 1 1 2 2 1 3 0 0
8 4 5 6 7 2 9 1 3
10 1
0 0 2 1 1 0 0 3 4
8 3 9 7 2 6 5 4 1
10 1
0 1 0 0 4 2 5 5 1
7 3 6 2 8 5 9 4 1
10 1
0 0 0 3 1 5 5 0 5
9 4 6 8 3 2 5 7 1
10 1
0 1 0 3 4 0 1 3 6
2 9 5 8 4 7 3 6 1
10 1
0 0 0 0 1 5 1 7 4
2 3 9 4 8 7 5 6 1
10 1
0 0 2 0 0 4 5 5 6
8 9 5 1 3 7 2 4 6
10 1
0 1 1 0 1 5 4 3 5
9 8 2 3 4 7 5 6 1
10 1
0 0 0 1 4 2 3 7 2
9 8 5 1 4 7 3 6 2
10 1
0 0 1 0 2 2 4 5 8
9 3 1 6 8 7 4 2 5
10 1
0 0 2 3 2 1 0 1 2
5 9 8 7 4 3 6 2 1
10 1
0 0 2 1 1 3 4 0 0
6 5 9 3 8 7 4 2 1
10 1
0 1 1 0 4 2 3 0 5
8 7 9 2 6 5 3 1 4
10 1
0 0 0 2 0 2 1 1 5
3 7 9 8 6 1 4 2 5
