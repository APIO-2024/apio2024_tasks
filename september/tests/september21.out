7ckgnn4wyi495puj3ibqf81dqvapyv6b
9
9
8
7
8
9
9
9
9
9
8
7
8
9
9
9
9
5
8
9
38
38
38
38
37
89
998
99
39
38
38
39
37
8
9
5
6
9
9
8
9
9
8
9
8
5
7
8
8
9
9
8
9
